pkg/controller/config/config_controller.go

```diff
@@ -412,6 +429,13 @@ func (r *ReconcileConfig) validatePipeline(req reconcile.Request, cfg *op.Config
 		return reconcile.Result{}, err
 
 	}
+
+	version := getComponentVersion(r.pipeline, flag.PipelineControllerName, "pipeline.tekton.dev/release")
+	if err = r.updateStatus(cfg, op.ConfigCondition{Code: op.AppliedPipelineVersion, Version: version}); err != nil {
+		log.Error(err, "failed to apply pipeline version")
+		return reconcile.Result{}, err
+	}
+
```

Comment:
shall we move this into `applyPipeline` or into `validatePipeline`

Reply:
this is already in validatePipeline